versions.json

```diff
@@ -0,0 +1,21 @@
+{
+  "versions": [
+    {
+      "date": "2022-01-27",
+      "protoc_version": "20.0-rc1",
+      "lts": false,
+      "languages": {
+        "cpp": "3.20.0-rc1",
+        "csharp": "3.20.0-rc1",
+        "java": "3.20.0-rc1",
+        "javascript": "3.20.0-rc1",
```

Comment:
If we're moving towards a word, where the submodules can ideally be owned by their own team, would it make sense to have a versions.json file within each language (directory/repo) which we support?

Reply:
Good question. I think `versions.json` is specifically to accommodate the current world, where multiple languages are bundled into `protoc`.

If/when we "unbundle" certain languages, so they can be in their own repo and their own codegen plugin, they will no longer need a `versions.json`. There can be a single release version for the whole repo, stored wherever it's most convenient/native for that repo/language. There will be no need to correlate a version across languages, so `versions.json` will not be necessary.